modules/decline/src/standalone/Smithy4sSimpleStandaloneCli.scala

```diff
@@ -34,14 +35,16 @@ import smithy4s.decline.util.PrinterApi
   *   parameters if you need program-wide config that doesn't fit into a specific operation's input
   *   (operation inputs are mapped to Opts automatically by all Smithy4sCli implementations).
   */
-class Smithy4sSimpleStandaloneCli[Alg[_[_, _, _, _, _]], Op[_, _, _, _, _]](
-    impl: Opts[smithy4s.Monadic[Alg, IO]]
+class Smithy4sSimpleStandaloneCli[Alg[_[_, _, _, _, _]], Op[_, _, _, _, _], F[
```

Comment:
So, this is kind of rendered useless by not extending `IOApp` anymore : it doesn't have a main method ... therefore I'm not too sure about this PR. We could have a `decline-effect` module that would re-create the CommandIOApp UX, otherwise this class ought to go and the documentation should showcase an example of using CommandIOApp in combination with the polymorphic construct.

Reply:
Why is it useless, its a pure convenience that handles Entrypoint creation and printer default.
I can switch to a factory method in companion object, but I still see value.
As far as docs are concerned , we currently have IOApp documented . I think CommandIOApp is not necessary as we are providing a command , one can call CommandIOApp.run and pass in the Command , but I think thats a separate concern